tests/nosetests/pyanaconda_tests/module_source_hmc_test.py

```diff
@@ -40,6 +40,11 @@ def type_test(self):
         """Test the type of SE/HMC."""
         self.assertEqual(SOURCE_TYPE_HMC, self.interface.Type)
 
+    @patch("pyanaconda.modules.payloads.source.hmc.hmc._", side_effect=lambda x: x)
+    def description_test(self, translation_mock):
```

Comment:
It would be good to add these tests to all sources.

Reply:
OK, will do.